runtime/j9vm/exports.cmake

```diff
@@ -413,6 +413,13 @@ else()
 	)
 endif()
 
+if(NOT JAVA_SPEC_VERSION LESS 21)
+	jvm_add_exports(jvm
```

Comment:
We should also stop exporting the old functions (but that could wait for a future pull request).
```
		JVM_VirtualThreadMountBegin
		JVM_VirtualThreadMountEnd
		JVM_VirtualThreadUnmountBegin
		JVM_VirtualThreadUnmountEnd
```

Reply:
Defer to a future pr - https://github.com/eclipse-openj9/openj9/issues/16984#issuecomment-1481788205